ert/data/record/_record.py

```diff
@@ -252,20 +261,18 @@ def load_collection_from_file(
     if mime == "application/octet-stream":
         if is_directory:
             return RecordCollection(
-                records=(BlobRecord(data=path_to_bytes(file_path)),),
+                records=(BlobRecord(data=await make_tar(file_path)),),
                 ensemble_size=ensemble_size,
                 collection_type=RecordCollectionType.UNIFORM,
             )
         else:
-            with open(file_path, "rb") as fb:
```

Comment:
There's a reason for `fb` short for file binary (bad abbreviation): if you branch code for opening a file on whether the file is binary or not, and use one variable name (`filehandle`) for both branches, mypy will complain in a cryptic way. It will say that the binary file API isn't compatible with TextIO or something. 

Assuming you're not aware of this already.

Reply:
Not seen those complaints no. Changing `filehandle` to `filebinary`